src/Raven.Client/Exceptions/ExceptionDispatcher.cs

```diff
@@ -231,26 +231,29 @@ private static ExceptionSchema GetExceptionSchema(HttpStatusCode code, Blittable
 
         private static async Task<BlittableJsonReaderObject> GetJson(JsonOperationContext context, HttpResponseMessage response, Stream stream)
         {
+            var memoryStream = new MemoryStream();
```

Comment:
I'm wondering if it would not be better to use `context.CheckoutMemoryStream` here? Thoughts? @grisha-kotler @ayende

Reply:
done